sklearn/tree/export.py

```diff
@@ -68,6 +68,8 @@ def _color_brew(n):
 class Sentinel(object):
     def __repr__(self):
         return '"tree.dot"'
+
```

Comment:
We usually don't want to make unrelated changes as it makes your work harder to review. In this case I'm fine with it

Reply:
Pep8 suggested that!